content/pages/playbook/design/visual.html

```diff
@@ -29,7 +29,7 @@
 
           <dl class="card">
             <dt>Headings: Roboto Slab, Source Sans Pro</dt>
-            <p>Content headings (headings 1 and 2 are reserved for the vets.gov header and breadcrumb navigation)</p>
+            <dt>Note that headings 1 and 2 are reserved for the vets.gov header and breadcrumb navigation</dt>
```

Comment:
@raquelromano-gov this actually isn't true anymore (we use H1 for the page title, and breadcrumbs are now a `nav` element), so maybe we should remove that line altogether for and add Heading 1 and Heading 2 back in for the time being. @suzchap thoughts?

(FYI there's an issue in the Nebula backlog somewhere to revisit this hierarchy)

![image](https://user-images.githubusercontent.com/4106356/27302885-b1a54ab0-54ed-11e7-9773-e76c20ead816.png)